src/index.js

```diff
@@ -1,14 +1,20 @@
 import express from 'express';
+import bodyParser from 'body-parser';
 import { serve, setup } from 'swagger-ui-express';
 import swaggerJsdoc from 'swagger-jsdoc';
 import swaggerDefinition from './swagger';
+import router from './routes/signupRoutes';
 
 const app = express();
 
 const swaggerOptions = {
   customSiteTitle: 'The Spinners',
   explorer: true,
 };
+app.use(bodyParser.json());
```

Comment:
There is no need to use body-parser. Express now comes with `express.json` and `express.urlencoded` functions.

Reply:
@higustave-ops ☝️